packages/flutter/lib/src/material/app.dart

```diff
@@ -628,15 +659,20 @@ class _MaterialAppState extends State<MaterialApp> {
         onGenerateInitialRoutes: widget.onGenerateInitialRoutes,
         onUnknownRoute: widget.onUnknownRoute,
         builder: (BuildContext context, Widget child) {
-          // Use a light theme, dark theme, or fallback theme.
+          // Resolve which theme to use based on brightness and high contrast.
           final ThemeMode mode = widget.themeMode ?? ThemeMode.system;
+          final ui.Brightness platformBrightness = MediaQuery.platformBrightnessOf(context);
```

Comment:
It shouldn't be necessary to qualify Brightness with `ui.`.

Reply:
Fixed